src/lib/Scenes/Fair/Components/FairBoothHeader.tsx

```diff
@@ -33,16 +34,17 @@ export const FairBoothHeader: React.SFC<Props> = ({
       </TouchableOpacity>
       <Spacer m={0.3} />
       <Sans weight="medium" size="3t">
-        {fairName} • Booth
+        {fairName}
       </Sans>
       <Spacer m={0.5} />
-      <Serif size="3t">Galleries</Serif>
-      <Spacer m={0.3} />
+      {boothLocation && <Serif size="3t">{boothLocation}</Serif>}
+      {boothLocation && <Spacer m={0.2} />}
```

Comment:
```jsx
{boothLocation &&
  <>
     <Serif size="3t">{boothLocation}</Serif>
     <Spacer m={0.2} />
  </>
}
```

Reply:
Also, doesn't look like `0.2` is in the Palette theme -- Should always pick the closest value from [`space`](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L116-L137), even if it slightly deviates from comps.